whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/lireditable/LirEditableInet6numAssignedPiAttributeValidationSpec.groovy

```diff
@@ -176,10 +177,11 @@ class LirEditableInet6numAssignedPiAttributeValidationSpec extends BaseLirEditab
         ack.summary.nrFound == 1
         ack.summary.assertSuccess(0, 0, 0, 0, 0)
         ack.summary.assertErrors(1, 0, 1, 0)
-        ack.countErrorWarnInfo(1, 0, 0)
+        ack.countErrorWarnInfo(2, 0, 0)
```

Comment:
modify resource, cannot change ripe-ncc mntner (mnt-routes) by lir

We shouldn't be changing the mnt-lower here.